src/_nested-list.scss

```diff
@@ -85,21 +85,10 @@ $button: #{$fd-namespace}-button;
     border-bottom: none;
 
     .#{$block}__group-header {
-      @include fd-nested-list-popover-border-top-radius();
-
       border-bottom: none;
     }
 
     .#{$block}__item {
-      &:first-child {
```

Comment:
assuming, nested list popover will always have popover header. so border rounding for first list item will not be needed.